js/types.ts

```diff
@@ -517,9 +518,41 @@ export interface AppState {
   tracks: TracksState;
 }
 
+/**
+ * Type definition of the music-metadata-browser module.
+ * Ref: https://github.com/Borewit/music-metadata-browser/blob/master/src/index.ts
+ */
```

Comment:
@durasj do you know an alternative to the redeclaration of the [modules interface](https://github.com/Borewit/music-metadata-browser/blob/master/src/index.ts)?